app/Http/Controllers/TaskCategoriesController.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use Illuminate\Support\Facades\Validator;
+
+use App\Models\TaskCategory;
+
+class TaskCategoriesController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $categoryList = TaskCategory::all();
+        return view('TaskCategory.index',compact('categoryList'));
+    }
+
+    public function changeStatus(Request $request) 
+    {
```

Comment:
Why do we need a status on category? Status is for a task/todo i.e. completed or not.

Reply:
It can be revert for category.